components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/java/org/wso2/carbon/identity/configuration/mgt/endpoint/exception/SearchConditionException.java

```diff
@@ -0,0 +1,12 @@
+package org.wso2.carbon.identity.configuration.mgt.endpoint.exception;
+
+/**
+ * Invalid search condition found.
+ */
+public class SearchConditionException extends RuntimeException {
```

Comment:
Missing license

Reply:
Fixed in e4e5d28c2d9caf6269b0ba6086fdb4203783ebbb